test/e2e/user_management_test.go

```diff
@@ -115,8 +115,11 @@ func (s *userManagementTestSuite) TestUserDeactivation() {
 			RequireConditions(ConditionSet(Default(), ApprovedByAdmin())...).
 			Execute().Resources()
 
+		// TODO remove once UserTier migration is completed
+		s.promoteToDefaultUserTier(hostAwait.Client, mur1)
+
```

Comment:
what is the purpose of promoting the user when it is deactivated manually?

Reply:
good point, it's not needed. Removed this and other cases in a6cd3440370feefacd130d319af1cd0fc90c9990